tsdx.config.js

```diff
@@ -15,19 +15,26 @@ module.exports = {
       case 'umd':
         delete config.external
         config.output.indent = false
-        config.plugins.push(
+        config.plugins.unshift(
           replace({
-            '// UMD-DEV-ONLY: ': ''
+            '// UMD-ONLY: ': '',
+            delimiters: ['', '']
           })
         )
-        config.plugins.push(
+        config.plugins.unshift(
           stripCode({
-            // Remove the `require()` import of RISI so we use the import statement
+            // Remove the `import` of RISI so we use the dynamic `require()` statement
             start_comment: 'START_REMOVE_UMD',
             end_comment: 'STOP_REMOVE_UMD'
           })
         )
         if (env === 'production') {
+          config.plugins.unshift(
+            stripCode({
+              start_comment: 'PROD_START_REMOVE_UMD',
+              end_comment: 'PROD_STOP_REMOVE_UMD'
```

Comment:
This `replace` instance allows for instantiation of `redux-immutable-state-invariant` in `development` builds only